src/KDSoapClient/moc_predefs.h

```diff
@@ -0,0 +1,389 @@
+#define __SSP_STRONG__ 3
```

Comment:
moc files are not supposed to be in version control.

Reply:
S#it. #188  submitted to solve these things forever